pgs_gist.sql.in

```diff
@@ -114,6 +117,7 @@ CREATE OPERATOR CLASS spoint
    OPERATOR  14 @ (spoint, spoly),
    OPERATOR  15 @ (spoint, sellipse),
    OPERATOR  16 @ (spoint, sbox),
+   OPERATOR	 17	<-> (spoint, spoint) FOR ORDER BY float_ops,
```

Comment:
Whitespace doesn't match the other `OPERATOR` lines surrounding this addition. The `17` should line up with the `16` on the previous line for starters.